apps/angular/projection/src/app/ui/card/card.component.ts

```diff
@@ -12,23 +18,37 @@ import { ListItemComponent } from '../list-item/list-item.component';
   standalone: true,
   imports: [NgIf, NgFor, ListItemComponent],
 })
-export class CardComponent {
-  @Input() list: any[] | null = null;
+export class CardComponent implements OnInit {
   @Input() type!: CardType;
-  @Input() customClass = '';
+  @Input() list: ListItem[] | null = null;
+  @Input() color = '';
 
   CardType = CardType;
+  imageUrl = '';
 
   constructor(
     private teacherStore: TeacherStore,
-    private studentStore: StudentStore
+    private studentStore: StudentStore,
+    private cityStore: CityStore,
   ) {}
 
+  ngOnInit(): void {
+    if (this.type === CardType.TEACHER) {
+      this.imageUrl = 'assets/img/teacher.png';
+    } else if (this.type === CardType.STUDENT) {
+      this.imageUrl = 'assets/img/student.png';
+    } else if (this.type === CardType.CITY) {
+      this.imageUrl = 'assets/img/city.png';
+    }
+  }
```

Comment:
this component is meant to be dumb. No logic should stay here. And the list will grow if you add more cardType. Look at @Output